src/Spago/Glob.purs

```diff
@@ -2,21 +2,28 @@ module Spago.Glob (gitignoringGlob) where
 
 import Spago.Prelude
 
+import Control.Alternative (guard)
+import Control.Monad.Maybe.Trans (MaybeT(..), runMaybeT)
+import Control.Monad.Trans.Class (lift)
 import Data.Array as Array
 import Data.Filterable (filter)
-import Data.Foldable (any)
+import Data.Foldable (any, fold)
 import Data.String as String
 import Data.String.CodePoints as String.CodePoint
+import Data.Traversable (traverse_)
 import Effect.Aff as Aff
 import Effect.Ref as Ref
 import Node.FS.Sync as SyncFS
 import Node.Path as Path
 import Record as Record
 import Type.Proxy (Proxy(..))
 
-type MicroMatchOptions = { ignore :: Array String }
+type MicroMatchOptions = { ignore :: Array String, include :: Array String }
```

Comment:
this change allows us to accumulate with `append`ing arrays instead of `||`ing functions.

Fun fact for those who may not be aware, if all of a record's fields are `Semigroup`, the record is `Semigroup` so you can `<>` these :)